tests/unit/web/test_session.py

```
@@ -0,0 +1,98 @@
+        sid = mgr.create(Path("/tmp/test.pdf"))
+        time.sleep(1.1)
+        session = mgr.get(sid)
+        assert session is None
+
+    def test_get_nonexistent_returns_none(self) -> None:
+        mgr = SessionManager()
+        assert mgr.get("nonexistent") is None
+
+    def test_update_structure(self) -> None:
+        mgr = SessionManager()
+        sid = mgr.create(Path("/tmp/test.pdf"))
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/test.pdf"

(S108)